docs/cli/block.rst

```diff
@@ -142,3 +142,7 @@ Block Commands
 .. click:: SoftLayer.CLI.block.set_note:cli
     :prog: block volume-set-note
     :show-nested:
+
```

Comment:
If there is any cloud.ibm.com/docs information on this feature, can you link it here?

Or at least elaborate a bit on some of the dangers of this command here.